data/model/gc.py

```diff
@@ -425,12 +425,8 @@ def _garbage_collect_manifest(manifest_id, context):
             .execute()
         )
 
-        # Subtract blobs from namespace/repo total after they've been deleted successfully.
-        # If the quota feature is disabled mark the total as stale
-        if features.QUOTA_MANAGEMENT:
-            subtract_blob_size(manifest.repository_id, manifest_id, blob_sizes)
-        else:
-            reset_backfill(manifest.repository_id)
+        # Subtract blob sizes if quota management is enabled
+        update_quota(manifest.repository_id, manifest_id, blob_sizes, "subtract")
```

Comment:
can you create a enum for the possible operations to `update_quota`

Reply:
yes